src/taskpane/server.js

```diff
@@ -1423,23 +1575,26 @@ async function sendExportEntriesExcel(entriesForExport, sheetData, sheet, sheetR
         log.parentId = getHierarchicalParentId(entriesForExport[i].indentPosition, log.entries);
       }
 
-      await manageSendingToSpira(entriesForExport[i], model.user, model.currentProject.id, artifact, fields, fieldTypeEnums, log.parentId)
+      //set the correct parentId artifact for Test Steps and property (needed for POST URL)
+      if (artifact.id == 2 && entriesForExport[i].hasOwnProperty('TestStepId')) {

```

Comment:
I am not sure what is going on here - why are you looking for an artifact with an ID of 2?
You should not hard code a field name here if you can avoid it. Try using data from the model to do whatever this code is doing more generically / dynamically